requirements.txt

```diff
@@ -1 +1 @@
--r https://dist.plone.org/release/5.2.9/requirements.txt
+-r https://dist.plone.org/release/5.2.10.1/requirements.txt
```

Comment:
It makes sense, but I have no strong opinion

Reply:
Makes sense - something like
`-r https://raw.githubusercontent.com/euphorie/Euphorie/master/requirements.txt` ?
Or referencing a tag like in base.cfg
`-r https://raw.githubusercontent.com/euphorie/Euphorie/14.2.0/requirements.txt` ?

Reply:
Check my comment - I was suggesting the latter.

Of course it would be even better if we didn't have multiple spots where we need to update the version number...

Reply:
Then let's do it like you suggested.

Reply:
BTW https://raw.githubusercontent.com/euphorie/Euphorie/14.2.0/requirements.txt still points to 5.2.10, not to 5.2.10.1... so it would be conceptually wrong.
Let's not waste more energy and keep the changes to the minimum.
What @thet did is already fine with me and consistent to what there was before.